rpc/jsonrpc/receipts/receipts_generator.go

```diff
@@ -148,7 +148,18 @@ func (g *Generator) GetReceipt(ctx context.Context, cfg *chain.Config, tx kv.Tem
 	txnHash := txn.Hash()
 
 	//if can find in DB - then don't need store in `receiptsCache` - because DB it's already kind-of cache (small, mmaped, hot file)
-	var receiptFromDB *types.Receipt
+	var receiptFromDB, receipt *types.Receipt
+
+	defer func() {
+		if dbg.Enabled(ctx) {
+			log.Info("[dbg] ReceiptGenerator.GetReceipt",
+				"txNum", txNum,
+				"txHash", txnHash.String(),
+				"receiptFromDB", receiptFromDB,
```

Comment:
`receiptFromDB` - logging of receipt itself is not useful - because we can see it in `json response`.